tests/test_benchmarks.py

```diff
@@ -127,11 +127,16 @@ def test_data():
 
 
 @pytest.mark.benchmark(group='load-data.toml')
-def test_load_data_toml(benchmark):
+def test_loads_data_toml(benchmark):
     data_toml_str = data_toml_path.read_text()
     # benchmark(rtoml.loads, data_toml_bytes, name='loads-bytes')
-    benchmark(rtoml.loads, data_toml_str, name='loads-str')
-    benchmark(rtoml.load, data_toml_path, name='load-path')
+    benchmark(rtoml.loads, data_toml_str)
+    # benchmark(rtoml.load, data_toml_path, name='load-path')
```

Comment:
if you want to change these, please remove the commented out lines.

Reply:
I didn't mean to change it, but it kept complaining that `rtoml.load(s)`  doesn't have `name` keyword argument. Which pytest-benchmark were you using? I don't find it in requirements.txt or test-requirements.txt.

Reply:
it's `pytest-speed` which is in requirements, pytest-speed is mostly backwards compatible with pytest-benchmark, except it allow multiple benchmarks per test - like this.